examples/inverse/plot_psf_ctf_label_leakage.py

```diff
@@ -0,0 +1,209 @@
+
+data_path = sample.data_path()
+subjects_dir = data_path + '/subjects'
+fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
+fname_inv = data_path + '/MEG/sample/sample_audvis-meg-oct-6-meg-fixed-inv.fif'
+
+# Read forward solution
+forward = mne.read_forward_solution(fname_fwd)
+# Convert to fixed source orientations
+mne.convert_forward_solution(
+    forward, surf_ori=True, force_fixed=True, copy=False)
+
+# Read inverse operator
+inverse_operator = read_inverse_operator(fname_inv)
+
+# Compute resolution matrices for MNE
+lambda2 = 1. / 3.**2
+method = 'MNE'
+rm_mne = make_inverse_resolution_matrix(forward, inverse_operator,
+                                        method=method, lambda2=lambda2)
+src = inverse_operator['src']
+del forward, inverse_operator  # save memory
+
+###############################################################################
+# Read and organise labels for cortical parcellation
+# --------------------------------------------------
+#
+# Get labels for FreeSurfer 'aparc' cortical parcellation with 34 labels/hemi
+labels = mne.read_labels_from_annot('sample', parc='aparc',
+                                    subjects_dir=subjects_dir)
+n_labels = len(labels)
+label_colors = [label.color for label in labels]
+
+# First, we reorder the labels based on their location in the left hemi
+label_names = [label.name for label in labels]
+
+lh_labels = [name for name in label_names if name.endswith('lh')]
+
+# Get the y-location of the label
+label_ypos = list()
+for name in lh_labels:
+    idx = label_names.index(name)
+    ypos = np.mean(labels[idx].pos[:, 1])
+    label_ypos.append(ypos)
+
+# Reorder the labels based on their location
+lh_labels = [label for (yp, label) in sorted(zip(label_ypos, lh_labels))]
+
+# For the right hemi
+rh_labels = [label[:-2] + 'rh' for label in lh_labels]
+
+###############################################################################
+# Compute point-spread function summaries (PCA) for all labels
+# ------------------------------------------------------------
+#
+# We summarise the PSFs per label by their first five principal components, and
+# use the first component to evaluate label-to-label leakage below.
+
+# Compute first SVD component across PSFs within labels
+# Note the differences in explained variance, probably due to different
+# spatial extents of labels
+n_comp = 5
+stcs_psf_mne, pca_vars_mne = get_point_spread(
+    rm_mne, src, labels, mode='pca', n_comp=n_comp, norm=None,
+    return_pca_vars=True)
+n_verts = rm_mne.shape[0]
+del rm_mne
+
+###############################################################################
+# We can show the explained variances of principal components per label. Note
+# how they differ across labels, most likely due to their varying spatial
+# extent.
+
+with np.printoptions(precision=1):
+    for [name, var] in zip(label_names, pca_vars_mne):
+        print('%s: %.1f%%' % (name, var.sum()))
+        print(var)
```

Comment:
```suggestion
        print(f'{name}: {var.sum():.1f}% {var}')
```
this condenses the output to one line for each label, and uses the f-string syntax instead of the (deprecated) %-replacement.